README.md

```diff
@@ -11,23 +11,28 @@ A fast and flexible Naive Bayes implementation for the JVM written in Kotlin.
   
 ## Usage
 
-```kotlin
-val model = Model().batchAdd(listOf(Update(
-                        inputs = Inputs( // Supports multiple feature types: text, categorical and gaussian.
-                                text = mapOf( 
-                                        "subject" to "Attention, is it true?", //features are named.
-                                        "body" to "Good day dear beneficiary. This is Secretary to president of Benin republic is writing this email ..." // multiple features of the same type have different names  
-                                ),
-                                categorical = mapOf(
-                                        "sender" to "WWW.@galaxy.ocn.ne.jp"
-                                )
+Get the latest artifact from [maven central](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.tradeshift%22%20a%3A%22blayze%22) 
+
+````java
+Model model = new Model().batchAdd(Lists.newArrayList(new Update( //Models are immutable
+                new Inputs( // Supports multiple feature types
+                        ImmutableMap.of( //Text features
+                                "subject", "Attention, is it true?", //features are named.
+                                "body", "Good day dear beneficiary. This is Secretary to president of Benin republic is writing this email ..." // multiple features of the same type have different names
+                        ),
+                        ImmutableMap.of(  //Categorical features
+                                "sender", "WWW.@galaxy.ocn.ne.jp"
                         ),
-                        outcome = "spam" // the outcome, in this case spam.
-                )
-        ))
-        
-val predictions: Map<Outcome, Double> = model.predict(Inputs(/*...*/)) // e.g. {"spam": 0.624, "ham": 0.376}
-```
+                        ImmutableMap.of(  //Gaussian features
+                                "n_words", 482.
+                        )
```

Comment:
Consider not using Guava for the example. 
I like Guava a lot, but for an example it might be more universal with plain JDK libs.